site/constants/index.ts

```diff
@@ -228,6 +230,7 @@ export const COOKIES_CONSEQUENCE_NETWORK_ERRORS = "cdd-consequence-network-error
 export const COOKIES_CONSEQUENCE_STOPS_ERRORS = "cdd-consequence-stops-errors";
 export const COOKIES_CONSEQUENCE_SERVICES_ERRORS = "cdd-consequence-services-errors";
 export const COOKIES_LOGIN_ERRORS = "cdd-consequence-login-errors";
+export const COOKIES_CHANGE_PASSWORD_ERRORS = "cdd-consequence-change-password-errors";
```

Comment:
should this have 'consequence' in the string?

Reply:
done